lib/bolt/execution_result.rb

```diff
@@ -78,6 +78,23 @@ def values
       @result_hash.values
     end
 
+    # Expects to be called with a configured Puppet compiler or error.instance? will fail
```

Comment:
Comment is no longer relevant.

Reply:
I think it still suffers from the same problem since the execution result outlives the compiler. I intend to clean it up some when this class goes away.